numpy/core/include/numpy/_dtype_api.h

```diff
@@ -141,7 +141,7 @@ typedef struct {
 #define NPY_METH_contiguous_indexed_loop 8
 
 /* other slots are in order, so note last one (internal use!) */
-#define _NPY_NUM_DTYPE_SLOTS 8
+#define _NPY_NUM_DTYPE_SLOTS 21 + (1 << 10)
```

Comment:
Can we leave this at 8?  So that it is easier to bump it to allow contiguous slots to be set without much thought?

Reply:
Sure! Done.